doc/superbuild/source/developers/index.rst

```diff
@@ -12,22 +12,29 @@
    SUNDIALS Copyright End
    -----------------------------------------------------------------------------
 
-#########################
-SUNDIALS Developers Guide
-#########################
+****************
+Developers Guide
+****************
+
+The SUite of Nonlinear and DIfferential/ALgebraic equation Solvers (SUNDIALS) consists of the time
+integration packages CVODE, IDA, and ARKODE, the sensitivity analysis enabled variants CVODES and
+IDAS, and the nonlinear solver package KINSOL. This is the part of the SUNDIALS documentation is
+meant for SUNDIALS developers.
 
 .. toctree::
-   :numbered:
-   :maxdepth: 2
+   :maxdepth: 1
 
-   introduction/index
+   History
    getting_started/index
    style_guide/index
    development/index
    testing/index
    benchmarks/index
    pull_requests/index
    releases/index
-   packages/index
```

Comment:
Looks like this got lost in the merge. I added a new directory with ARKODE specific documentation in the full RHS PR.